docker-compose.yml

```diff
@@ -24,6 +24,7 @@ services:
     #   - ./data_sources.py:/app/caluma/extensions/data_sources.py
     #   - ./format_validators.py:/app/caluma/extensions/format_validators.py
     environment:
+      - ENV=dev
```

Comment:
This is already set [in the docker-compose.override.yml#L26](https://github.com/projectcaluma/caluma/blob/master/docker-compose.override.yml#L26). Value can be anything except `production`.

Reply:
Ah, right! @StephanH90 has the override file not been picked up in your case?

Reply:
In that case I'm not sure why that didn't work. I remember that when I made a fresh install of docker and a clone of the repo I got a bunch of warning along the lines of: "secret key needs to be set", etc.

However, I suspect that this was a problem with my local setup.